app/styling/styled-components/registry.tsx

```diff
@@ -1,20 +1,23 @@
 'use client';
 
-import React from 'react';
+import React, { useState } from 'react';
 import { useServerInsertedHTML } from 'next/navigation';
-import { useStyledComponentsRegistry } from '#/lib/styling';
+import { ServerStyleSheet, StyleSheetManager } from 'styled-components';
 
-export default function StyledComponentsRegistry({
-  children,
-}: {
-  children: React.ReactNode;
-}) {
-  const [StyledComponentsRegistry, styledComponentsFlushEffect] =
-    useStyledComponentsRegistry();
+type ChildProps = { children: React.ReactNode };
+
+export default function StyledComponentsRegistry({ children }: ChildProps) {
```

Comment:
```suggestion
export default function StyledComponentsRegistry({
  children,
}: {
  children: React.ReactNode
}) {
```

Reply:
I don't have a strong personal preference, but just being consistent with elsewhere where we define types inline.